src/resolvers/Mutation/createAdvertisement.ts

```diff
@@ -6,7 +6,7 @@ export const createAdvertisement: MutationResolvers["createAdvertisement"] =
   async (_parent, args, _context) => {
     // Creates new Ad.
     const createdAd = await Advertisement.create({
```

Comment:
`const createdAdvertisement = await ....`

try to be explicit and precise in naming, there's no harm in writing long self-descriptive names